app/imports/components/Users/AdminPanel/Panel.jsx

```diff
@@ -0,0 +1,49 @@
+import RaisedButton from 'material-ui/RaisedButton';
+import FontIcon from 'material-ui/FontIcon';
+import { Card, CardTitle } from 'material-ui';
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import UsersList from './UsersManagement/UsersList';
+import ProjectList from './UsersManagement/ProjectList';
+
+export default class Panel extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { option: true };
+        this.changeOption = this.changeOption.bind(this);
+    }
+
+    changeOption() {
+        this.setState({ option: !this.state.option });
+    }
+
+    render() {
+        return (
+            <div>
+                <Card>
+                    <CardTitle
+                        title="Retro Tool- Admin Panel"
+                        subtitle="Admin"
+                    />
+                    <RaisedButton
+                        onTouchTap={this.changeOption}
+                        label="Manage Users"
+                        secondary
```

Comment:
I don't know what about this, because we have two secondary buttons and none primary button. It's required? I think that standard button will be sufficient. What do you think about it?